javascript/javascript1/week1/homework/housey-pricey.js

```diff
@@ -0,0 +1,29 @@
+
+
+console.log("======Exercise Housey pricey======");
+
+// Peter's house price
+let houseCost = 2500000;
+let volumeInMeters = 8 * 10 * 10;
+let gardenSizeInM2 = 100;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log("Peter's house price is:", housePrice);
+if (housePrice < 2500000) {
+    console.log("By paying", houseCost.toLocaleString()+" Peter is overpaying for the house.");
```

Comment:
You get a bit too many `"` here.
```suggestion
    console.log("By paying " + houseCost.toLocaleString() + " Peter is overpaying for the house.");
```